libs/deal-ticket/src/hooks/use-calculate-slippage.ts

```diff
@@ -22,7 +22,7 @@ const useCalculateSlippage = ({ marketId, order }: Props) => {
     variables,
     throttleMilliseconds: 5000,
   });
-  const { data: market } = useDataProvider<SingleMarketFieldsFragment, never>({
+  const { data: market } = useDataProvider<Market, never>({
```

Comment:
Don't get it why this change is for? `Market` type is returned by market**s**Provider.
marketProvider returns `SingleMarketFieldsFragment` type. Now these types maybe are similar, but in the future they can get some discrepancies

Reply:
makes sense. I'm reverting to `SingleMarketFieldsFragment`